code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -661,15 +633,13 @@
 	if(M.a_intent in list(INTENT_HARM,INTENT_DISARM,INTENT_GRAB))
 		src.harmedBy(M)
 
-// also maybe we've just had environmental damage, who knows
 /mob/living/critter/flock/drone/TakeDamage(zone, brute, burn, tox, damage_type, disallow_limb_loss)
 	..()
 	src.check_health()
 	if (brute <= 0 && burn <= 0 && tox <= 0)
 		return
 	var/prev_damaged = src.damaged
 	if(!isdead(src) && src.is_npc)
-		// if we've been damaged a new stage, call it out
```

Comment:
useful comment

Reply:
I left part of it, think saying was already explained by the say()